functions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once ('index.php');
+
+/*
+ * displays each film and its attributes (from the database) in a human readable way
+ *
+ * @param array which is an array containing arrays for each film
+ *
+ * return string displaying films and film details
+ */
+function displayFilms(array $array) : string {
```

Comment:
$films would be a better name for your this array

Reply:
updated to $flms